src/module/localstorage.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-cycle */
+import addedTask from './class';
+import displayList from './displaylist';
+
+const addToLocalStorage = () => {
+  const stringifyArray = JSON.stringify(addedTask.taskArray);
+  localStorage.setItem('storedTask', stringifyArray);
+};
+
+const getFromLocalStorage = () => {
+  const stringifyArray = localStorage.getItem('storedTask');
+  addedTask.taskArray = JSON.parse(stringifyArray);
+  displayList();
+  console.log(addedTask.taskArray);
+};
```

Comment:
In order to follow [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), please avoid committing console.log on the repo, it kills performance.